脚手架初始化时选择不使用dva时，项目运行报错。

1. 复现步骤：初始化时选择app模版，并且不勾选dva，勾选antd(这个应该不影响)。

2. 运行`npm start`后项目报错`AssertionError: Invalid key dva from plugin`，可以通过修改`.umirc.js`中`dva`为`true`或者删除app.js来修复。

3. 这个问题应该是umi的插件配置校验导致的，所以选择不使用dva生成的app.js文件中不应该有dva字段。

相关issue: [#1506](https://github.com/umijs/umi/issues/1506)

--- a/lib/generators/app/index.js
+++ b/lib/generators/app/index.js
@@ -39,7 +39,7 @@ class Generator extends BasicGenerator {
         if (isTypeScript) {
           if (f.endsWith('.js')) return false;
           if (!reactFeatures.includes('dva')) {
-            if (f.startsWith('src/models') && f === 'src/app.ts') return false;
+            if (f.startsWith('src/models') || f === 'src/app.ts') return false;
           }
           if (!reactFeatures.includes('locale')) {
             if (f.startsWith('src/locales') || f.includes('umi-plugin-locale')) return false;
@@ -47,7 +47,7 @@ class Generator extends BasicGenerator {
         } else {
           if (this.isTsFile(f)) return false;
           if (!reactFeatures.includes('dva')) {
-            if (f.startsWith('src/models') && f === 'src/app.js') return false;
+            if (f.startsWith('src/models') || f === 'src/app.js') return false;
           }
           if (!reactFeatures.includes('locale')) {
             if (f.startsWith('src/locales') || f.includes('umi-plugin-locale')) return false;

